site/profile/manifests/volumes.pp

```diff
@@ -94,7 +94,7 @@
     followsymlinks   => true,
   }
 
-  if $filesystem == 'xfs' and $quota {
+  if $filesystem == 'xfs' {
     $options = 'defaults,usrquota'
   } else {
     $options = ''
```

Comment:
lvm::volume does not allow `$options` to be an empty string.
https://github.com/puppetlabs/puppetlabs-lvm/blob/main/manifests/logical_volume.pp#L70

Reply:
What are the cases where "options" is an empty string?

Reply:
`$options` would also have been an empty string if `$fileystem` was set to `"ext4"` for example.